modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerSelfTest.java

```diff
@@ -218,7 +237,7 @@ public void testCreateTable() {
         assertEquals(manager.zone(ZONE_NAME, clock.nowLong()).id(), table.zoneId());
 
         // Validate newly created pk index
-        assertEquals(3L, pkIndex.id());
+        assertEquals(4L, pkIndex.id());
```

Comment:
Let's drop checking object ids.

Reply:
I think it is good idea to check ids, because this way we see that ids  are assigned correctly. Updating them from time to time is not a huge price to pay for that guarantee.

Reply:
What does correct id mean?
We may have different strategies to assign ids. They may be strictly-monotonic function or random function with low collisions or hash or smth else. Also, at catalog start, we may create a new object (e.g. information schema view) in future, so each time all these test have to be fixed.

I'd just left checks to validate the object is available by id, which it was created with, different objects has different ids.
Also, kind of ABA-problen test, that we never assign same id for equal objects, if we really don't want to.

Reply:
Fixed.